src/databricks/labs/ucx/source_code/notebook.py

```diff
@@ -38,6 +51,43 @@ def build_dependency_graph(self, parent: DependencyGraph):
         raise NotImplementedError()
 
 
+class PythonLinter(Linter):
```

Comment:
move this to astlinter.py ( -> `python_analysis.py`)

Reply:
done